server/pxf-hdfs/src/main/java/org/greenplum/pxf/plugins/hdfs/parquet/ParquetTypeConverter.java

```diff
@@ -66,6 +68,17 @@ public Object getValue(Group group, int columnIndex, int repeatIndex, Type type)
                 return group.getString(columnIndex, repeatIndex);
             }
         }
+
+        @Override
+        public void addValueToArray(Group group, int columnIndex, int repeatIndex, PrimitiveType primitiveType, PgArrayBuilder pgArrayBuilder) {
+            Object value = getValue(group, 0, 0, primitiveType);
```

Comment:
is there ever a case where the col index or the repeat index value will not be 0?

nit:
```suggestion
                                        Object value = getValue(group, 0 /*col index*/, 0 /*repeat index*/, primitiveType);
```